converter/src/test/java/gov/cms/qpp/conversion/decode/NullReturnDecoder.java

```diff
@@ -8,7 +8,7 @@
 import gov.cms.qpp.conversion.model.TemplateId;
 
 @Decoder(TemplateId.NULL_RETURN)
-public class NullReturnDecoder extends QppXmlDecoder {
+class NullReturnDecoder extends QppXmlDecoder {
```

Comment:
You made a decoder (versus a test) non-public. Was this intentional?

Reply:
Accident. Thanks for pointing that out.